### DISCUSSION TITLE
Dump an entire table records

### DISCUSSION THREAD

**Discussion by gauravkrp** [directus/directus#16625] [Feature Requests]
I was importing some 40K records into a table via a CSV file but due to some errors (first 413 and then 504), the table ended up having 80K records, i.e, double import. 

Is there a way to bulk delete entire records (like dump the entire table) either via API or from the UI itself without touching the database?

**Comment by br41nslug** [comment]
You can delete items using the API https://docs.directus.io/reference/items.html#delete-multiple-items but you may need to batch that as 80k at once will probably error.

**Reply by gauravkrp** [reply]
Yup, that's one way around, another could be to create a new table but the option to dump entries without (losing the schema) is what I'm hoping for.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
